crates/accelerate/src/unitary_compose.rs

```diff
@@ -217,24 +197,25 @@ fn _ind(i: usize, reversed: bool) -> usize {
     }
 }
 
-/// For equally sized matrices, ``left`` and ``right``, check whether all entries are close
-/// by the criterion
-///
-///     |left_ij - right_ij| <= atol + rtol * right_ij
-///
-/// This is analogous to NumPy's ``allclose`` function.
-pub fn allclose(
+pub fn mm2q(
```

Comment:
Maybe write this out as `matmul_2q` or something a bit more descriptive? It took me a sec to understand what this meant at first. Or just a docstring to say what it is.